network/network_test.go

```diff
@@ -1467,11 +1465,11 @@ func TestPeerAliasesDisconnect(t *testing.T) {
 	)
 	id2 := ids.ShortID(hashing.ComputeHash160Array([]byte(ip2.IP().String())))
 
-	vdrs := validators.NewManager(0,
-		validators.WithValidator(id0, 1),
-		validators.WithValidator(id1, 1),
-		validators.WithValidator(id2, 1),
-	)
+	//vdrs := validators.NewDefaultSet(constants.CostonID)
```

Comment:
Why was this kept commented out? Should be removed unless there's a good reason for it IMO.